WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/breakpoint/BreakpointChangeType.java

```diff
@@ -0,0 +1,46 @@
+package nl.tudelft.watchdog.core.logic.breakpoint;
+
+import com.google.gson.annotations.SerializedName;
+
+/** The different types of changes to a breakpoint. */
+public enum BreakpointChangeType {
+	/** Breakpoint has been enabled. */
+	@SerializedName("en")
+	ENABLED,
+
+	/** Breakpoint has been disabled. */
+	@SerializedName("ds")
+	DISABLED,
+	
+	/** Hit count added on breakpoint. */
```

Comment:
What is the semantics of this? I would assume bp never hit = has no hit count.
bp hit = adds hit_count?
Would this be easier for our analysis purposes to just drop the first event and look at hc_changed then? I might be totally wrong here.